packages/plugin-redirects/package.json

```diff
@@ -30,14 +30,14 @@
   "author": "dev@payloadcms.com",
   "license": "MIT",
   "peerDependencies": {
-    "payload": "^0.18.5 || ^1.0.0 || ^2.0.0"
+    "payload": "^0.18.5 || ^1.0.0 || ^2.0.0",
```

Comment:
should this catch 3.x as well?

Reply:
yea I can open a PR against beta for that